verticapy/core/vdataframe/_plotting.py

```diff
@@ -1614,149 +1805,228 @@ def barh(
         ).draw(**kwargs)
 
     @save_verticapy_logs
-    def hist(
+    def pie(
         self,
-        by: Optional[str] = None,
         method: str = "density",
```

Comment:
You took out method for vDataFrame so you can take it out here as well. 
If you want to keep it, then you may want to add the "PlottingMethod"

Reply:
I think it is wrong. Nothing was erased, it is a Github wrong formatting.